src/devices/src/virtio/net/test_utils.rs

```diff
@@ -302,12 +302,19 @@ pub fn write_element_in_queue(net: &Net, idx: usize, val: u64) -> result::Result
     Ok(())
 }
 
+#[allow(clippy::cast_sign_loss)] // okay for testing
```

Comment:
The assertion can be removed and the conversion replaced with `Ok(u64::try_from(net.queue_evts[idx].as_raw_fd()).unwrap())`.